pages/users/[urlusername].tsx

```diff
@@ -0,0 +1,141 @@
+import {useContext, useState} from "react";
+import banner from "public/okkoro_banner.png";
+import Image from 'next/image'
+import {collection, getDocs, where} from "@firebase/firestore";
+import {getFirestore} from "firebase/firestore";
+import {query} from "@firebase/database";
+import {docToJSON} from "../../lib/firebase";
+import {useRouter} from "next/router";
+import {Button, Col, Container, Row} from "react-bootstrap";
+import {getRecommendation} from "../../lib/recommendations"
+import MovieList from "../../components/MovieList";
+import {UserContext} from "../../lib/context";
+import ProfileMovieList from "../../components/ProfileMovieList";
+
+
+export default function Profile() {
+
+    const {username} = useContext(UserContext);
+
+    const router = useRouter()
+    const {urlusername} = router.query
+    // console.log(urlusername)
+
+    return (
+
+        <div className="row bg-green">
+            {/*{urlusername == username ? <SignedInProfile urlusername={urlusername}/> : <WrongProfile/>}*/}
```

Comment:
zombie?

Reply:
Yes this code is not useful for now however it will be used for next sprint when we incorporate authorization because currently, anyone can access anyone else's account, all it is checking is the URL.

Reply:
understood 👍